support/client/lib/mil-sym/cws.js

```diff
@@ -5493,6 +5683,11 @@ define( function(){
                   "symbolType": "MultiSegmentPolygon",
                   "validModifiers": [ "hostile" ]
             },
+            "TACGRP.MOBSU.OBST.MNE.MCLST": {
+                  "symbolID": "G*MPOMC---****X",
+                  "tag": "MINE CLUSTER",
+                  "symbolType": "SimpleLine"
```

Comment:
Is a `MINE CLUSTER` really supposed to be a line?  I would have thought that would be an area.  But what do I know?

Reply:
It's funny.  the spec actually specify it as a 2-point line that then results in kind of a half-moon.

Reply:
Peculiar.  :smile:  If I make the line long, the half-moon is truncated on top.  It becomes two parallel lines that are connected by a subtle arc.  Is that how it's supposed to look.